Tasks/VsTestV2/distributedtest.ts

```diff
@@ -99,9 +99,16 @@ export class DistributedTest {
             if (this.inputDataContract.TestSelectionSettings.TestSelectionType.toLowerCase() !== 'testassemblies') {
                 sourceFilter = ['**\\*', '!**\\obj\\*'];
             }
-
+            const taskProps: { [key: string]: any; } = {MiniMatchLines: sourceFilter.length};
+            ci.publishTelemetry('TestExecution','MinimatchFilterPerformance', taskProps);
+            var start = new Date().getTime();
             const sources = tl.findMatch(this.inputDataContract.TestSelectionSettings.SearchFolder, sourceFilter);
             tl.debug('tl match count :' + sources.length);
+            var end = new Date().getTime();
+            var timeTaken = end - start;
+            tl.debug('Time taken in milli seconds '+timeTaken);
```

Comment:
tl.debug(`Time taken for applying the minimatch pattern to filter out the sources ${timeTaken} ms`);

nitpick: small change in wording and use string interpolation